dev-python/openpyxl/openpyxl-2.6.2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1
+
+MY_COMMIT="3640394bff97"
+
+DESCRIPTION="Pure python reader and writer of Excel OpenXML files"
+HOMEPAGE="https://openpyxl.readthedocs.io/en/stable/"
+SRC_URI="https://bitbucket.org/${PN}/${PN}/get/${PV}.tar.bz2 -> ${P}.tar.bz2"
```

Comment:
Have you asked upstream if they could provide tests in PyPI tarballs? That would be more convenient than changing the commit hash for each bump.

Reply:
Hello @sbraz 
I haven't asked them, and I'm not sure about what you mean. Could you please elaborate?

Reply:
I'm pretty sure that the reason Bitbucket is used instead of PyPI is the lack of tests in PyPI tarballs. It's a classic :)
I opened an upstream issue: https://bitbucket.org/openpyxl/openpyxl/issues/1308/include-tests-in-pypi-tarballs

Reply:
Hello
I understand that upstream are against the inclusion of tests in PyPI. So in this case no change is needed?

Reply:
Yeah, can you just add a comment above SRC_URI linking to my issue?

Reply:
Done. Is it enough?

Reply:
Yeah, that's fine :) I wonder if there's a nice way to properly set `S` without specifying the commit hash which is a pain.

Reply:
Well, I though about wildcard - but doesn't work, and is very unsecure in my idea.
Another option is like in [vcs-snapshot.eclass](https://github.com/gentoo/gentoo/blob/master/eclass/vcs-snapshot.eclass) - which is unsupported in `EAPI=7`, but it look as over complication.

Reply:
Yeah I was just asking @mgorny if he had plans to make this eclass EAPI-7-compatible since it's rather simple.

Reply:
@arthurzam not sure where that comment with the link went, I'm re-adding it before I merge the PR.